articles/quickstart/webapp/aspnet-core/_includes/_login.md

```diff
@@ -113,11 +118,11 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 }
 ```
 
-### Obtaining an Access Token for calling an API
+## Obtain an Access Token for Calling an API
 
-You may want to call an API from your MVC application, in which case you need to obtain an `access_token` which was issued for the particular API you want to call. In this case you will need to pass an extra `audience` parameter containing the API Identifier to the Auth0 authorization endpoint. 
+If you want to call an API from your MVC application, you need to obtain an access token issued for the API you want to call. To obtain the token, to the Auth0 authorization endpoint, pass an additional `audience` parameter containing the API identifier.
```

Comment:
>  To obtain the token, to the Auth0 authorization endpoint, pass an additional `audience` parameter containing the API identifier.

This reads very strangely. It should be

>  To obtain the token, pass an additional `audience` parameter containing the API identifier to the Auth0 authorization endpoint.

Because the `audience` parameter is passed to the endpoint. The way you are wording it, it sounds as though you are obtaining an access token to the endpoint. Which is not correct.